packages/itwinui-css/src/table/table.scss

```diff
@@ -15,7 +15,8 @@
   overflow: hidden;
   // Fix for Firefox columns misalignment
   @supports not (overflow: overlay) {
-    overflow-y: scroll;
+    scrollbar-gutter: stable;
```

Comment:
Interesting. Does this affect behaviors in other browsers though?

Reply:
it's inside `@supports not (overflow: overlay)`. Chrome and Safari do support `overflow: overlay` so it will not affect them.

Reply:
Oh wow that's another interesting thing to know. Thanks